security/list-users.php

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Login</title>
+</head>
+<body>
+<a href="index.php"> Take me home, country roads ... </a> <br>
+<?php
+session_start();
+if (!isset($_SESSION["email"])) {
+    die("Only authenticated users are allowed");
+}
+
+include "db-connection.php";
+$conn = openCon();
+
+
+
+$sql = "SELECT * from person where email = '" . $_SESSION["email"] . "';";
```

Comment:
This is not protected against sql injection. The correct way to use prepared statements is to bind the actual values of the parameters after the preparation of the statement. Currently if there is some malicious symbols in the email your application may break (at least). Please check here for a reference implementation: https://github.com/tancheto/WEB2020-php/blob/4a5142909392a6d4ca90e939d13abd33844707a0/hello-security/login-controller.php